src/functions_framework/__init__.py

```diff
@@ -31,9 +33,18 @@
 )
 from google.cloud.functions.context import Context
 
+
+from cloudevents.sdk.event import v1
+from cloudevents.sdk import marshaller
```

Comment:
```suggestion
import cloudevents.sdk
import cloudevents.sdk.event
```

Also these imports should be moved up with the other 3rd party imports (flask/werkzeug)